lib/internal/vm.js

```diff
@@ -66,6 +79,12 @@ function getHostDefinedOptionId(importModuleDynamically, hint) {
   return Symbol(hint);
 }
 
+/**
+ * Registers a dynamically imported module for customization.
+ * @param {string} referrer - The path of the referrer module.
+ * @param {import('internal/modules/esm/utils').ImportModuleDynamicallyCallback} importModuleDynamically - The
```

Comment:
`importModuleDynamically` can be a known host option id.

```suggestion
 * @param {import('internal/modules/esm/utils').ImportModuleDynamicallyCallback | symbol} importModuleDynamically - The
```